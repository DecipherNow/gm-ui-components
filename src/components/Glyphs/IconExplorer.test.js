import React from "react";
import { shallow } from "enzyme";

import IconExplorer from "./IconExplorer.svg";

describe("IconExplorer", () => {
  it("matches snapshot", () => {
    const aIconExplorer = shallow(<IconExplorer />).find("IconExplorer.svg");
    expect(aIconExplorer).toMatchSnapshot();
  });
});