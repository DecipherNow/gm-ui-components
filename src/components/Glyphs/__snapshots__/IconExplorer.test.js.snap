// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconExplorer matches snapshot 1`] = `<IconMock />`;
